index.js

```diff
@@ -8,6 +8,71 @@ const checkWIPModule = require('./lib/checkWipDraftPR');
 
 const whitelistedAccounts = (
   (process.env.WHITELISTED_ACCOUNTS || '').toLowerCase().split(','));
+const checksWhitelist = {
+  'oppia-android': {
+    'opened': ['cla-check'],

```

Comment:
Per my comment below, I suggest moving each of these event types (opened/reopened, etc.) to their own constants, then you can add a brief comment for each one explaining when it happens. Then, in all places currently referencing 'opened' we would reference the constant, instead. I think that'll help make the code a bit more readable and easier to change in the future.

Reply:
Done.